src/main/java/openfasttrack/importer/tag/TagImporter.java

```diff
@@ -82,15 +86,21 @@ public void runImport()
     private void processLine(final int lineNumber, final String line)
     {
         final Matcher matcher = this.tagPattern.matcher(line);
+        int counter = 0;
         while (matcher.find())
         {
             this.listener.beginSpecificationItem();
-            final SpecificationItemId id = SpecificationItemId.parseId(matcher.group(1));
+            final String generatedName = this.fileName + ":" + lineNumber + "-" + counter;
+            final SpecificationItemId generatedId = SpecificationItemId.createId(matcher.group(1),
+                    generatedName, 0);
```

Comment:
Now I generate the name using the CRC32 from filename, line number, counter and covered id.